sensor-validate.cpp

```diff
@@ -1,28 +1,44 @@
 #include "sensor-validate.h"
 
-bool _give_me_a_good_name(double value, double nextValue, double maxDelta) {
-  if(nextValue - value > maxDelta) {
+bool isSensorReadingsHaveSuddenJumps_b(double currentValue, double nextValue, double maxDelta) {
```

Comment:
Name of this function suggests that it will return 1 when the sensor reading has sudden jump. But actually it is the other way round.

Reply:
@sudeeprp : I'll update the functionality accordingly. Thank You 👍